README.rst

```diff
@@ -20,33 +20,68 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Function
+========
+
+``pgshovel`` is a system which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL database out of the database and into an ordered stream of transaction-aware mutation messages. These messages can then be consumed by 3rd party systems as needed. Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
+
+At Disqus ``pgshovel`` is actively being developed to replicate changes from our master PostgreSQLdatabase - which contains the core commenting and other relational data - into our data warehousing system. There is further planned work to use the mutation stream to power a denormalized objects database, offering highly available low-latency access to object data.
```

Comment:
Heh okay.  I can make "the database" change though.

On Thu, Sep 3, 2015 at 10:25 PM, ted kaemming <notifications@github.com>
wrote:

> In README.rst
> <https://github.com/disqus/pgshovel/pull/25#discussion_r38722497>:
>
> > @@ -20,33 +20,68 @@ configurations remain synchronized with the state of all databases in the
> >  cluster. PgQ is used for buffering mutation records until they are collected by
> >  the relay and forwarded to their final destination.
> >
> > +Function
> > +========
> > +
> > +``pgshovel`` is a system which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL database out of the database and into an ordered stream of transaction-aware mutation messages. These messages can then be consumed by 3rd party systems as needed. Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
> > +
> > +At Disqus ``pgshovel`` is actively being developed to replicate changes from our master PostgreSQLdatabase - which contains the core commenting and other relational data - into our data warehousing system. There is further planned work to use the mutation stream to power a denormalized objects database, offering highly available low-latency access to object data.
>
> I like the hyphens as a way to break up that sentence, and also give a
> sense of what we store in PostgreSQL. Were you thinking using commas or
> parenthesis nothing?
>
> This is mostly just grammatical nitpicking. If you want to make it a
> parenthetical that's is fine, but I'd replace "which" with "this database"
> in that case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/disqus/pgshovel/pull/25/files#r38722497>.
>